app/models/settings.rb

```diff
@@ -44,6 +44,7 @@ def update(attributes)
 
     super(attributes)
   end
+  alias update_attributes update
```

Comment:
Do we really need this? Is it too bad to call the proper method?

Reply:
No, that's removed in the later cleanup commit: https://github.com/3scale/porta/pull/3456/commits/eb2346546cd7560f201510c26e8ec38b5fd52b6d#diff-6f750351adedd314c4f977fe861185a6f544b045e3d4969724786216a7dce38b